src/Simple.php

```diff
@@ -652,6 +653,13 @@ public function process($serial)
             // track the time needed for the import in seconds
             $endTime = microtime(true) - $startTime;
 
+            if ($ase instanceof ApplicationFinishedException) {
```

Comment:
Wouldn't it make sense that we create a new catch block like `... } catch (ApplicationFinishedException) { ...` to make it more transparent?

Reply:
true